source/agora/node/Ledger.d

```diff
@@ -830,6 +864,7 @@ unittest
     blocks = ledger.getBlocksFrom(0).take(10).array;
     assert(blocks.length == 3);
     assert(blocks[2].header.height == 2);
+
```

Comment:
Extra blank line.

Reply:
Ah, it is unittest code lines. I'm doing now.